content/doc/book/system-administration/monitoring.adoc

```diff
@@ -11,7 +11,7 @@ ifndef::env-github[:imagesdir: ../../resources/managing]
 :toc: left
 endif::[]
 
-= Monitoring Jenkins
+== Monitoring Jenkins
```

Comment:
The build fails with the message:

> [2023-03-03T08:48:57.570Z] WARNING: Missing title for /doc/book/system-administration/monitoring/index.html

The file needs a level 1 heading to provide the title of the page.
```suggestion
= Monitoring Jenkins
```